detectron2/modeling/roi_heads/keypoint_head.py

```diff
@@ -120,7 +120,11 @@ def keypoint_rcnn_inference(pred_keypoint_logits, pred_instances):
     pred_keypoint_logits = pred_keypoint_logits.detach()
     keypoint_results = heatmaps_to_keypoints(pred_keypoint_logits, bboxes_flat.detach())
     num_instances_per_image = [len(i) for i in pred_instances]
-    keypoint_results = keypoint_results[:, :, [0, 1, 3]].split(num_instances_per_image, dim=0)
+    # slicing multiple dimensions with sequences is not supported by torchscript
+    # https://github.com/pytorch/pytorch/issues/43943
+    keypoint_results = keypoint_results.index_select(dim=2, index=torch.tensor([0, 1, 3])).split(
+        num_instances_per_image, dim=0
+    )
```

Comment:
```
import torch

def func(x: torch.Tensor):
    idx = torch.tensor([0, 1, 3])
    return x[:, :, idx]

f = torch.jit.script(func)
```
the above works in my local test. Could you try this? It's a smaller change from the original

Reply:
This works well:)